manticore/platforms/linux.py

```diff
@@ -1397,7 +1394,7 @@ def sys_exit_group(self, error_code):
         #self.procs[procid] = None
         logger.debug("EXIT_GROUP PROC_%02d %s", procid, error_code)
         if len(self.running) == 0 :
-            raise ProcessExit(error_code)
+            raise Exception('Process exited correctly. Code: {}'.format(error_code))
```

Comment:
Yeah :( .. ideas?
Import and raise state.TerminateState ?
Make a PlatformException for all the platforms and raise that?